src/components/RegistrationRow.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import {TextInput, CheckBox, Dropdown} from '../columns/RegistrationColumns';
+
+import {TEXT, CHECKBOX, DROPDOWN} from '../columns/types';
+
+const defaultRowStyle = {
+  display: 'flex',
+  flexWrap: 'wrap',
+  justifyContent: 'flex-start',
+  padding: '10px',
+  button: {
+    alignSelf: 'flex-end',
+    flexGrow: '2',
+    color: '#070',
+    fontWeight: 'bold'
+  }
+};
+
+class RegistrationRow extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rowData: {}
+    };
+    this.addRow = this.addRow.bind(this);
+    this.deleteRow = this.deleteRow.bind(this);
```

Comment:
Gå for en av kode konvensjonene.. Enten bind alle funksjoner, eller ingen. sendValue er ikke bindet.